src/codegen.cpp

```diff
@@ -84,6 +84,7 @@ CodeGen *codegen_create(Buf *root_source_dir, const ZigTarget *target) {
         g->libc_lib_dir = buf_create_from_str("");
         g->libc_static_lib_dir = buf_create_from_str("");
         g->libc_include_dir = buf_create_from_str("");
+        g->zig_std_dir = buf_create_from_str("");
```

Comment:
Yes, but I'm planning to do a 'playground' project which will include a Zig binary, so people can start using it easily. And I don't know where would they put the folder! I preferred making a patch rather than doing some trickery like compiling Zig in `/tmp`, or forcing the user to install it in a privileged place like `/opt`